olive/passes/onnx/transformer_optimization.py

```diff
@@ -127,6 +127,14 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
             "input_int32": PassConfigParam(
                 type_=bool, default_value=False, description="Whether int32 tensors will be used as input."
             ),
+            "attention_op_type": PassConfigParam(
```

Comment:
This config param is not used since the actual implementation gets it from `optimization_options` which itself if a config param.

Reply:
removed